.github/workflows/build-test-ubuntu20.yml

```
@@ -44,6 +44,31 @@ jobs:
           # not realy needed
           CMAKE_C_COMPILER: ${{ matrix.c-compiler }}
 
+      - name: Collect Timings
+        run: |
+          inFolder=./build/${{matrix.config}}/
+          outFolder=./build/${{matrix.config}}/${{ fromJSON('["Clang_11", "GCC_10"]')[matrix.compiler == 'GCC 10'] }}/Timings
+          mkdir -p $outFolder
+          folderList=(MRCommonPlugins mrviewerpy MRTest MRViewerApp MRViewer mrmeshpy meshconv mrmeshnumpy MRMesh)
+          for folder in ${folderList[*]}
+          do
+            mkdir $outFolder/$folder
+            cp $inFolder/$folder/compile_timings.txt $outFolder/$folder/compile_timings.txt
+            cp $inFolder/$folder/link_timings.txt $outFolder/$folder/link_timings.txt
+          done
+          currentWd=`pwd`
+          cd $outFolder/..
+          zip Ubuntu20_${{matrix.config}}_${{ fromJSON('["Clang_11", "GCC_10"]')[matrix.compiler == 'GCC 10'] }}_timings.zip $outFolder
+          cd $currentWd
+
+      #Save timing in artifact
+      - name: Upload timing
+        uses: actions/upload-artifact@v2
+        with:
+          name: Ubuntu20_${{matrix.config}}_${{ fromJSON('["Clang_11", "GCC_10"]')[matrix.compiler == 'GCC 10'] }}_timings
```

This action allows to upload different files to the same artifact
```suggestion
          name: TimeLogs
```

---

fixed